src/main/scala/chisel3/Driver.scala

```diff
@@ -3,18 +3,14 @@
 package chisel3
 
 import chisel3.internal.ErrorLog
-import chisel3.internal.firrtl.Converter
+import chisel3.internal.firrtl.{Converter, _}
```

Comment:
lolwut

Reply:
Yeah. This was an unwanted IntelliJ import rewrite that slipped through my import review. Will fix.